ComplementaryData/physiology/biomassComposition_Cofactor_Ion.tsv

```diff
@@ -0,0 +1,20 @@
+s_3880[c]	Ca(2+) 	2.17E-04	40.08	ion	DOI: 10.22161/ijeab/2.2.10
+s_3778[c]	chloride 	2.55E-03	35.45	ion	DOI: 10.22161/ijeab/2.2.11
+s_4019[c]	Cu2(+) 	6.59E-04	63.55	ion	DOI: 10.22161/ijeab/2.2.12
+s_0924[c]	iron(2+) 	3.04E-05	55.84	ion	DOI: 10.22161/ijeab/2.2.13
+s_3801[c]	Mn(2+) 	2.73E-03	54.94	ion	DOI: 10.22161/ijeab/2.2.14
+s_3822[c]	Zn(2+) 	7.48E-04	65.39	ion	DOI: 10.22161/ijeab/2.2.15
+s_1373[c]	potassium 	3.63E-03	39.1	ion	DOI: 10.22161/ijeab/2.2.16
+s_1437[c]	sodium 	3.97E-03	22.99	ion	DOI: 10.22161/ijeab/2.2.17
+s_4013[c]	Mg(2+) 	0.001242543	24.3	ion	DOI: 10.22161/ijeab/2.2.18
+s_1467[c]	sulphate	0.02	96.06	ion	Forster2003
+s_1203[c]	NADH 	0.000788462	663.43	cofactor	YMDB
+s_1198[c]	NAD 	0.001	663.43	cofactor	YMDB
+s_1212[c]	NADPH 	7.69E-05	745.4209	cofactor	YMDB
+s_1207[c]	NADP(+) 	6.54E-05	744.4129	cofactor	YMDB

```

Comment:
For these 4 mets, it should be indicated the specific link with the information, not just the database.